conjure-java-core/build.gradle

```
@@ -103,7 +103,7 @@ dependencies {
     integrationInputImplementation 'com.google.errorprone:error_prone_annotations'
     integrationInputImplementation 'com.google.guava:guava'
     integrationInputImplementation 'com.palantir.conjure.java.api:errors'
-    integrationInputImplementation 'com.palantir.dialogue:dialogue-target:3.99.0'
+    integrationInputImplementation 'com.palantir.dialogue:dialogue-target:3.100.0'
```

Why is there a version number here? Shouldn't this use GCV version.props?

```suggestion
    integrationInputImplementation 'com.palantir.dialogue:dialogue-target'
```